chcemvediet/apps/inforequests/forms/obligee_action.py

```diff
@@ -684,18 +526,267 @@ def post_transition(self):
+
+        return res
+
+class ContainsAppealInfo(ObligeeActionStep):
+    label = _(u'inforequests:obligee_action:ContainsAppealInfo:label')
+    text_template = u'inforequests/obligee_action/texts/contains_appeal_info.html'
+    global_fields = [u'disclosure_level']
+    post_step_class = WasItAccepted
+
+    def add_fields(self):
+        super(ContainsAppealInfo, self).add_fields()
+
+        self.fields[u'disclosure_level'] = forms.TypedChoiceField(
+                label=u' ',
+                coerce=int,
+                choices=(
+                    (Action.DISCLOSURE_LEVELS.FULL,
+                        _(u'inforequests:obligee_action:ContainsAppealInfo:full')),
+                    (Action.DISCLOSURE_LEVELS.PARTIAL,
+                        _(u'inforequests:obligee_action:ContainsAppealInfo:partial')),
+                    (Action.DISCLOSURE_LEVELS.NONE,
+                        _(u'inforequests:obligee_action:ContainsAppealInfo:none')),
+                    ),
+                widget=forms.RadioSelect(),
+                )
+
+class IsItAppealDecision(ObligeeActionStep):
+    label = _(u'inforequests:obligee_action:IsItAppealDecision:label')
+    text_template = u'inforequests/obligee_action/texts/is_appeal_decision.html'
+
+    def add_fields(self):
+        super(IsItAppealDecision, self).add_fields()
+
+        self.fields[u'is_decision'] = forms.TypedChoiceField(
+                label=u' ',
+                coerce=int,
+                choices=(
+                    (1, _(u'inforequests:obligee_action:IsItAppealDecision:yes')),
+                    (0, _(u'inforequests:obligee_action:IsItAppealDecision:no')),
+                    ),
+                widget=forms.RadioSelect(),
+                )
+
+    def post_transition(self):
+        res = super(IsItAppealDecision, self).post_transition()
+
+        if not self.is_valid():
+            branch = self.wizard.values.get(u'branch', None)
+            if branch.can_add_disclosure_refusal_advancement_extension:
+                res.next = CanAddDisclosureRefusalAdvancementExtension
+            else:
+                res.next = ContainsAppealInfo
+        elif self.cleaned_data[u'is_decision']:
+            res.next = ContainsAppealInfo
+        else:
+            res.next = CanAddDisclosureRefusalAdvancementExtension
+
+        return res
+
+class CanAddRemandmentAffirmationReversion(ObligeeActionStep):
+
+    def pre_transition(self):
+        res = super(CanAddRemandmentAffirmationReversion, self).pre_transition()
+        branch = self.wizard.values.get(u'branch', None)
+
+        if self.wizard.email:
+            res.next = CanAddDisclosureRefusalAdvancementExtension
+        elif not branch:
+            res.next = CanAddDisclosureRefusalAdvancementExtension
+        elif branch.can_add_remandment_affirmation_reversion:
+            res.next = IsItAppealDecision
+        else:
+            res.next = CanAddDisclosureRefusalAdvancementExtension
 
         return res
 
```

Comment:
Pred classami `CanAddRemandmentAffirmationReversion` az `InvalidReversion` je komentar "Post Appeal". Dopln nejaky komentar aj pred `class IsItConfirmation`, kedze `IsItConfirmation` nie je "Post Appeal" krok.

Kedze "Pre Appeal" su az za "Post Appeal" krokmi, tak tieto uvodne kroky oznac napriklad ako "Confirmation and Clarification Request". Alebo tak nejak.